Refactor components and componentized common design patterns.

jeromeZestyIO opened the pull request:
refactor(website common components): refactor components and componentized common design patterns.

Components mapping reference:
https://docs.google.com/spreadsheets/d/135NfOovFtsI-ipf0fvHfe39r7OLfpqBCCKIGoNFdtAA/edit#gid=0

merged on dev:
https://zesty-dev-website-m3rbwjxm5q-uc.a.run.app/

Page urls:
/
/why-zesty/
/solutions/headless-cms/
/solutions/hybrid-cms/
/solutions/digital-experience-platform/
/integrations/nextjs-cms/
/mindshare/customer-stories/
/news/
/content-resources/
/pricing/
/agency-partners/partner-program-benefits/
/demos/
/demos/video/
/ppc/content-management-system/
/use-case/marketers/
/use-case/for-developers/

jeromeZestyIO committed:
refactor(common layout components): refactor compontents and componentized common design patterns

jeromeZestyIO committed:
refactor(website common components): refactor compontents and componentized common design patterns

jeromeZestyIO committed:
refactor(common layout components): refactor compontents and componentized common design patterns

jeromeZestyIO committed:
refactor(common layout components): refactor compontents and componentized common design patterns

jomarmontuya committed:
refactor(features and logo components): refactor fillercontents

jomarmontuya committed:
refactor(features and logo components): refactor fillercontents

jeromeZestyIO committed:
fix(common components): fix component fallback contents

jeromeZestyIO committed:
refactor(layout components): fix component fallback contents

jeromeZestyIO committed:
Merge remote-tracking branch 'origin/refactor' into refactor

jeromeZestyIO committed:
refactor(image with contents cta component): create ImageWithContentsCta component

darwin808 committed:
test(homepage): added test

darwin808 reviewed (approved):
LGTM

darwin808 committed:
Merge branch 'stage' into refactor

Katiemoser commented:
@jeromeZestyIO - a couple things:

Image on homepage hero looks squished
![Screenshot 2022-12-15 at 8 04 50 AM](https://user-images.githubusercontent.com/101665566/207911848-964db6d8-d26c-4d03-97a1-21b9174dc79f.png)

I believe most of these should have H3s instead of H2s
![Screenshot 2022-12-15 at 8 05 54 AM](https://user-images.githubusercontent.com/101665566/207911983-9188bc65-df00-49d9-8ab9-7a20e5d147d8.png)

jeromeZestyIO commented:
Hi @Katiemoser,

I've made the edits on the page:
- fixed homepage hero image
- Changed H2s to H3s
- Adjusted spacing between header and the content

pr link: https://github.com/zesty-io/website/pull/1455

Thanks!

![Zesty-io-Simplify-digital-Maximize-results-](https://user-images.githubusercontent.com/115722439/207929107-c0da2f42-8691-48d4-9089-be4a6dea8aba.png)
